p2p/sentry/sentry_multi_client/sentry_multi_client.go

```diff
@@ -356,6 +361,7 @@ func NewMultiClient(
 		maxBlockBroadcastPeers:            maxBlockBroadcastPeers,
 		disableBlockDownload:              disableBlockDownload,
 		logger:                            logger,
+		getReceiptsActiveGoroutineNumberController: semaphore.NewWeighted(1),
```

Comment:
first part of variable name looks enough informative - without `Controller` suffix

Reply:
done